setup.py

```diff
@@ -35,7 +35,7 @@
 ]
 
 REQUIREMENTS_INSTALL = [
-    "openscm-units>=0.1.1",
+    "openscm-units>=0.2,<0.3",
```

Comment:
Is there a particular reason this isn't futureproof, or are you just preventing another update breaking things?

Reply:
Mainly the latter (just preventing another update breaking things). 

The longer reason is that I help maintain openscm-units, so I know that when it goes to 0.3, it'll be because the API is changed in some way. This just makes sure that we have a chance to check things all work before we declare silicone to be openscm-units 0.3 compatible.